src/components/KTreeList/KTreeItem.vue

```diff
@@ -1,45 +1,39 @@
 <template>
-  <a
-    class="k-tree-item"
+  <div
```

Comment:
question: why is this a `div` rather than a `button` or an `a` tag?

---

If you use a `div` we'll need `tabindex="0"` as well

Reply:
Adding `tabindex`. Changed it from `a` to `div` because it's definitely not a link and on the docs site it causes issues (page jump).

Reply:
any reason not to use a button?

Reply:
No particular reason apart from "doesn't feel right" since this is draggable items that aren't really buttons.. do you think it should be a button?

Reply:
@kaiarrowood since you were the one working on this originally, any reason it is not using list tags like `ul` and `li`?

Reply:
I think a list structure makes the most sense TBH unless there's a strong reason not to use it.

Reply:
Kai went through the comments on the original PR adding this component and couldn't find any arguments against using `ul` here so I'll work on refactoring this tomorrow.

Reply:
Hi team! So I pushed my changes implementing the ordered list structure, also recorded a Loom walking through the resulting tree structure and outlining couple small caveats that came with this change:
https://www.loom.com/share/013dcb4129b44626b898782a9d5c5134?sid=e66bd70d-db41-4c2a-bf37-9959e83373f8

Reply:
Update: the bug with multiple drag zone blue lines is fixed now.

Reply:
Update: we decided to leave the idea utilizing the list element structure because it's hard to keep the HTML structure semantic, especially after user reordering when a deeply nested item can end up being the first-level item and so on.